### ISSUE TITLE
"requests" integration is actually "httplib"

### ISSUE THREAD

**Issue by bshaffer** [census-instrumentation/opencensus-python#125]
Following the instructions for [integrating requests](https://github.com/census-instrumentation/opencensus-python#requests) results in the following error:

```
WARNING:opencensus.trace.config_integration:Failed to integrate module: requests, supported integrations are httplib, mysql, postgresql, pymysql, requests, sqlalchemy
```

This is because the docs say `request` but it should be `httplib`

**Comment by wkiser** [comment]
Both requests and httplib integrations are supported (just tested it out). Are you sure that you have the requests package installed? It looks like there is a generic error for failing to add an integration.

**Comment by bshaffer** [comment]
Ahh, that must be it. Thank you!

I would then suggest changing this issue to be a more descriptive error message.

**Comment by liyanhui1228** [comment]
Yes both the `requests` and `httplib` integrations are supported, and the possible reason could be `requests` module not installed like @wkiser said. Yeah thanks for the feedback, I was intending to not break user's application when integrating with the libraries so I made the exception generic, will make a change to output more meaningful error message.

**Comment by bshaffer** [comment]
I've installed and imported the `requests` module but I am still getting the error

**Comment by wkiser** [comment]
Hey @bshaffer , can you paste a gist of the code you are trying to execute and the output of `pip freeze`? Are you running from source or from a package installed from pypi?

**Comment by bshaffer** [comment]
@wkiser 

Here is my code: https://github.com/bshaffer/faceswap-app/blob/v2/worker/worker.py

And here is the output of pip freeze: https://gist.github.com/bshaffer/9014f0ae21a06fca0005994780d1f6a1
